### PR TITLE
Portofolio transfer

### PR DESCRIPTION
**Pull request by Bogdy87**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,375 +1,375 @@
...
-    <script src="script.js"></script>
-  </body>
-</html>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
```
Comment (jason-vasilev) [inline index.html:7]: I'd recommend to write your name for better search engine optimization. Remember the <title> tag shows as the titles in the Google searches. Something as "Bogdan Dobinda Front-end Developer", could be better.

File: index.html [hunk 2]
```diff
@@ -1,375 +1,375 @@
...
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="mediaqueries.css" />
+  </head>
+  <body>
+    <nav id="desktop-nav">
+      <div class="logo"><img src="./assets/logo3.png" alt="PrivatLogo" /></div>
```
Comment (jason-vasilev) [inline index.html]:
```suggestion
      <img class="logo" src="./assets/logo3.png" alt="PrivatLogo" />

```

You can have 1 less div here by moving the class to the image. The CSS would have to be updated, too.

File: index.html [hunk 3]
```diff
@@ -1,375 +1,375 @@
...
+      <div>
+        <ul class="nav-links">
+          <li><a href="#about">About</a></li>
+          <li><a href="#experience">Experience</a></li>
+          <li><a href="#projects">Projects</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
```
Comment (jason-vasilev) [inline index.html:21]: Would recommend wrapping this in `<nav>` tag which will help indicate to website crawlers and people using asssitive technology that it's the main navigation.
Here are some examples from MDN - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav
and W3Schools - https://www.w3schools.com/TAgs/tag_nav.asp

File: index.html [hunk 4]
```diff
@@ -1,375 +1,375 @@
...
+      </div>
+    </nav>
+    <nav id="hamburger-nav">
+      <div class="logo"><img src="./assets/logo3.png" alt="PrivatLogo"</div>
```
Comment (jason-vasilev) [inline index.html:25]: I'm glad to see you use alt (alternative text tag). You could change the value to something more descriptive eg. "Logo for Creative Design by Bogdy Alex"

File: index.html [hunk 5]
```diff
@@ -1,375 +1,375 @@
...
+      <div class="hamburger-menu">
+        <div class="hamburger-icon" onclick="toggleMenu()">
+          <span></span>
+          <span></span>
+          <span></span>
+        </div>
```
Comment (jason-vasilev) [inline index.html:31]: This won't be accessible when using keyboard navigation through Tab key (might be a bit advanced concept at this stage). Using a semantic tag as `<button>` could solve it by default. As a quick fix you could add `role="button"`.

File: index.html [hunk 6]
```diff
@@ -1,375 +1,375 @@
...
+        <div class="menu-links">
+          <li><a href="#about" onclick="toggleMenu()">About</a></li>
+          <li><a href="#experience" onclick="toggleMenu()">Experience</a></li>
+          <li><a href="#projects" onclick="toggleMenu()">Projects</a></li>
+          <li><a href="#contact" onclick="toggleMenu()">Contact</a></li>
+        </div>
```
Comment (jason-vasilev) [inline index.html:37]: I like that you are implementing a event for hiding the mobile menu on interaction.

File: index.html [hunk 7]
```diff
@@ -1,375 +1,375 @@
...
+      </div>
+    </nav>
+    <section id="profile">
+      <div class="section__pic-container">
+        <img src="./assets/profile-pic.png" alt="John Doe profile picture" />
+      </div>
```
Comment (jason-vasilev) [inline index.html:43]: Just as on the logo - you could spare youself a `<div>` tag and move the class and styling directly to the image. Remember to update the `alt`

File: index.html [hunk 8]
```diff
@@ -1,375 +1,375 @@
...
+      <div class="section__text">
+        <p class="section__text__p1">Hello, I'm</p>
+        <h1 class="title">Bogdan Dobinda</h1>
+        <p class="section__text__p2">Frontend Developer</p>
```
Comment (jason-vasilev) [inline index.html:47]: I love the idea and execution of this.

File: index.html [hunk 9]
```diff
@@ -1,375 +1,375 @@
...
+        <div class="btn-container">
+          <button
+            class="btn btn-color-2"
+            onclick="window.open('./assets/resume-example.pdf')"
+          >
+            Download CV
+          </button>
```
Comment (jason-vasilev) [inline index.html:54]: You can also make it as an anchor tag and skip JavaScript.
```suggestion
          <a 
            class="btn btn-color-2"
            href="./assets/resume-example.pdf"
          >
            Download CV
          </a>

```

File: index.html [hunk 10]
```diff
@@ -1,375 +1,375 @@
...
+          <button class="btn btn-color-1" onclick="location.href='./#contact'">
+            Contact Info
+          </button>
```
Comment (jason-vasilev) [inline index.html:57]: Could an `<a>` tag with `href` work here instead of <button>?

File: index.html [hunk 11]
```diff
@@ -1,375 +1,375 @@
...
+        </div>
+        <div class="about-details-container">
+          <div class="about-containers">
+            <div class="details-container">
+              <img
+                src="./assets/experience.png"
+                alt="Experience icon"
+                class="icon"
+              />
+              <h3>Experience</h3>
```
Comment (jason-vasilev) [inline index.html:94]: The heading order is inconsistent. This is a very advanced concept so you don't need to pay much attention to it at this stage. Even digital agency websites don't follow it often.
Here is a brief explanation - https://dequeuniversity.com/rules/axe/4.8/heading-order and much better here - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#accessibility_concerns

File: index.html [hunk 12]
```diff
@@ -1,375 +1,375 @@
...
+        </div>
+      </div>
+      <img
+        src="./assets/arrow.png"
+        alt="Arrow icon"
+        class="icon arrow"
+        onclick="location.href='./#experience'"
+      />
+    </section>
+    <section id="experience">
```
Comment (jason-vasilev) [inline index.html:126]: Glad to see you use `sections`. Makes it very clear and easy to break down the content on the page.

File: index.html [hunk 13]
```diff
@@ -1,375 +1,375 @@
...
+      <nav>
+        <div class="nav-links-container">
+          <ul class="nav-links">
+            <li><a href="#about">About</a></li>
+            <li><a href="#experience">Experience</a></li>
+            <li><a href="#projects">Projects</a></li>
+            <li><a href="#contact">Contact</a></li>
+          </ul>
+        </div>
+      </nav>
```
Comment (jason-vasilev) [inline index.html:371]: Nice that you use <nav> tag. Could you move the `nav-links-container` class up and remove the `div`?

File: mediaqueries.css [hunk 14]
```diff
@@ -1,103 +1,103 @@
...
-  }
-  .title {
-    font-size: 2rem;
-  }
-  .text-container {
-    text-align: justify;
-  }
-}
+@media screen and (max-width: 1400px) {
+  #profile {
```
Comment (jason-vasilev) [inline mediaqueries.css:2]: Generally it's preferred to use IDs for selecting elements with JavaScript, and class names for styling. However, it's nothing critical, as it works anyways and no functionality is broken.

**Review by jason-vasilev — APPROVED** [review]
Good code and practices. Few recommendations and comments, but you are on a very good track.

File: index.html (hunk 3 above)
Comment (Bogdy87) [reply index.html:21]: I have changed "div" with "nav" for this listed items. I hope it is fine , and not a problem using too many "nav" tags. it's not affecting nothing until now.

File: index.html (hunk 5 above)
Comment (Bogdy87) [reply index.html:31]: Good idea. I have changed and also CSS for this.

File: index.html (hunk 13 above)
Comment (Bogdy87) [reply index.html:371]: Done! Thanks.

File: mediaqueries.css (hunk 14 above)
Comment (Bogdy87) [reply mediaqueries.css:2]: I can't make so much difference between ID and classes (they do the same thing). I am not confident in using them  properly where and when, but I will take in consideration your comment . Thanks.

**Pull request closed without merging**